app/src/main/java/com/github/sdp/tarjetakuna/database/UserRTDB.kt

```diff
@@ -151,5 +152,31 @@ class UserRTDB(database: Database) { //Firebase.database.reference.child("users"
         return future
     }
 
+    /**
+     * Push the location of the user to the database.
+     */
+    fun pushUserLocation(userUID: String, location: Coordinates) {

```

Comment:
would be nice to return a Task or a future that complete when those 2 actions are done